<?xml version="1.0"?>
<launch>
    <!-- Physical Parameters-->
    <param name="/navigation/lqr/prepare_track/min_width"  value="3.0"/>
    <param name="/navigation/lqr/handler/track_width"  value="5.0"/>
    <param name="/navigation/lqr/optimize_track/vehicle_width"  value="1.5"/>

    <!-- Track Prep Parameters-->
    <param name="/navigation/lqr/prepare_track/k_reg"   value="3"/>
    <param name="/navigation/lqr/prepare_track/s_reg"  value="5"/>
    <param name="/navigation/lqr/prepare_track/stepsize_prep"  value="1.0"/>
    <param name="/navigation/lqr/prepare_track/stepsize_reg"  value="3.0"/>

    <!-- Raceline Parameters-->
    <param name="/navigation/lqr/raceline/stepsize_interp"  value="1.0"/>

    <!-- Car Boundaries-->
    <param name="/navigation/lqr/handler/safety_margin"  value="0.5"/>
    <param name="/navigation/lqr/optimize_track/curvature_boundaries"  value="0.12"/>

    <!-- Toggle Plotting-->
    <param name="/navigation/lqr/handler/plot"  value="1"/>
</launch>
